templates/development.php

```diff
@@ -276,17 +276,43 @@
         document.getElementById('dark-mode').onclick = function(e) {
             e.preventDefault();
 
-            document.body.classList.add('dark-theme');
+            enableDarkTheme();
 
             setCookie('yii-exception-theme', 'dark-theme');
         }
 
         document.getElementById('light-mode').onclick = function(e) {
             e.preventDefault();
 
-            document.body.classList.remove('dark-theme');
+            enableLightTheme();
+
+            setCookie('yii-exception-theme', 'light-theme');
+        }
+
+        var darkModeMediaQuery = window.matchMedia('(prefers-color-scheme: dark)');
```

Comment:
Isn't it better not to use JS for the CSS purpose?

Reply:
Then selectors will be duplicated, because we need to override the state with a button
```
.dark-theme header {
    background: #2e2e2e;
}
@media (prefers-color-scheme: dark) {
    body:not(.light-theme) header {
        background: #2e2e2e;
    }
}
```

Reply:
> Then selectors will be duplicated, because we need to override the state with a button

Perfect, still better that a piece of JS code. Would you do it?

Reply:
Yes. I can try using css variables instead of duplicating